submissions/DawoduRahmon/HTML/product-page.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
Remove this file from your PR @Dawodurahmonn You have index.html already.

Reply:
ma, pls ma i only have one index.html in my folder.
kindly put me through on what to do here ma.

Reply:
Exactly, you have a product page already which is your index.html. These extra product pages you added are not needed.
 Still on that same branch, update your changes.
Add your changes (git add .).
Commit your changes (git commit  -m  "commit message").
Finally, Push your change (git push origin branch_name)
@Dawodurahmonn